src/current/_includes/releases/v23.1/v23.1.11.md

```diff
@@ -0,0 +1,141 @@
+## v23.1.11
+
+{{ site.data.alerts.callout_danger }}
```

Comment:
What do you think about using a Note (`site.data.alerts.callout_info`) instead of a Warning?

Reply:
Updated to `callout_info` per Style Guide.